docs/api.md

```diff
@@ -637,7 +637,7 @@ Page is guaranteed to have a main frame which persists during navigations.
 
 #### page.pdf(options)
 - `options` <[Object]> Options object which might have the following properties:
-  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided, the PDF won't be saved to the disk.
+  - `path` <[string]> The file path to save the PDF to. If `path` is a relative path, then it is resolved relative to [current working directory](https://nodejs.org/api/process.html#process_process_cwd). If no path is provided, the PDF won't be saved to the disk. If the path does not exist, the method throws an error.
```

Comment:
oh wait, this is confusing now: the path technically doesn't exist here:
```js
page.screenshot({path: 'screenshot.png'})
```
Why do we want to do anything about this statement? Was there a confusing somewhere on the bugtracker?

Reply:
From https://github.com/GoogleChrome/puppeteer/pull/798